<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>crud_materiel</class>
 <widget class="QMainWindow" name="crud_materiel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chef de depot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>820</width>
      <height>550</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Gestion du Matériel</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>801</width>
        <height>531</height>
       </rect>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Ajouter Materiel</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>110</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Ref Matériel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>170</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom Matériel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>226</x>
          <y>230</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Nombre</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>110</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nom">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>170</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_prenom">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>230</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_ajouter">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>250</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-60</y>
          <width>821</width>
          <height>571</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>label_5</zorder>
       <zorder>label</zorder>
       <zorder>label_2</zorder>
       <zorder>label_3</zorder>
       <zorder>lineEdit_id</zorder>
       <zorder>lineEdit_nom</zorder>
       <zorder>lineEdit_prenom</zorder>
       <zorder>pb_ajouter</zorder>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Afficher le Matériel</string>
       </attribute>
       <widget class="QTableView" name="tabetudiant">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>90</y>
          <width>400</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-60</y>
          <width>821</width>
          <height>571</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_6">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>40</y>
          <width>251</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Recherche</string>
        </property>
       </widget>
       <zorder>label_6</zorder>
       <zorder>tabetudiant</zorder>
       <zorder>lineEdit_id_6</zorder>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Chercher</string>
       </attribute>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>60</y>
          <width>211</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Rechercher par référence :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_5">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>60</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>60</y>
          <width>191</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>60</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Recherche par nom :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>160</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Ref Matériel</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="nom">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>220</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="ref">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>160</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>220</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom Matériel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>176</x>
          <y>280</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Nombre</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="nbr">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>280</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_supprimer_3">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>190</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_supprimer_4">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>260</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-60</y>
          <width>821</width>
          <height>571</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>490</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Ref Matériel</string>
        </property>
       </widget>
       <zorder>label_8</zorder>
       <zorder>label_11</zorder>
       <zorder>lineEdit_id_5</zorder>
       <zorder>comboBox</zorder>
       <zorder>label_12</zorder>
       <zorder>label_15</zorder>
       <zorder>nom</zorder>
       <zorder>ref</zorder>
       <zorder>label_14</zorder>
       <zorder>label_13</zorder>
       <zorder>nbr</zorder>
       <zorder>pb_supprimer_3</zorder>
       <zorder>pb_supprimer_4</zorder>
       <zorder>label_16</zorder>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>0</y>
        <width>101</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Déconnexion</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
